types/openlayers/index.d.ts

```diff
@@ -8484,7 +8485,7 @@ export namespace source {
          * @param style Layer style.
          * @api stable
          */
-        setStyle(style: (ol.style.Style | ol.style.Style[] | ol.StyleFunction)): void;
+        setStyle(style: (ol.style.Style | ol.style.Style[] | ol.StyleFunction | undefined | null)): void;
```

Comment:
OK but can you be coherent in the definition order? The previous one is `null | undefined`, this one is `undefined | null`. It's not just cosmetic, it may make maintenance easier.

Reply:
Sure, done.